src/app-layout/__integ__/constants.ts

```diff
@@ -4,3 +4,6 @@ export const viewports = {
   desktop: { width: 1200, height: 800 },
   mobile: { width: 400, height: 800 },
 };
+
+//must match the number in  '../../../lib/components/app-layout/visual-refresh/drawers';
+export const VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT = 2;
```

Comment:
Can we reuse the existing variable from `'../../../lib/components/app-layout/visual-refresh/drawers'`?

Reply:
I received many failures trying to import in the tests.  I timeboxed the resolution and would rather focus elsewhere

Reply:
It felt weird having a variable that is used in refresh tests for refresh-toolbar but it works